.travis.yml

```diff
@@ -1,40 +1,63 @@
-language: python
-sudo: false
-python:
-- '2.7'
-- '3.4'
-- '3.5'
+# System information
+os: linux
 addons:
   postgresql: "9.3"
+services:
+  - redis
 
-cache:
-  directories:
-  - $HOME/.pip-cache/
+language: python
+python: 3.5
```

Comment:
Is there any reason for choosing 3.5?

Reply:
It works -- every other version seems to have some issue with the outdated-and-pinned dependencies.

Reply:
Ah. Okay. Will keep a note to update this in future if needed.